app/src/main/java/fr/free/nrw/commons/customselector/ui/selector/CustomSelectorActivity.kt

```diff
@@ -74,12 +77,41 @@ class CustomSelectorActivity: BaseActivity(), FolderClickListener, ImageSelectLi
         }
 
         // Open folder if saved in prefs.
-        if(prefs.contains(FOLDER_ID)){
+        if(prefs.contains(FOLDER_ID)) {
             val lastOpenFolderId: Long = prefs.getLong(FOLDER_ID, 0L)
             val lastOpenFolderName: String? = prefs.getString(FOLDER_NAME, null)
             val lastItemId: Long = prefs.getLong(ITEM_ID, 0)
             lastOpenFolderName?.let { onFolderClick(lastOpenFolderId, it, lastItemId) }
         }
+
+            if(isImageGalleryEmpty())
+        {
+            val linearLayout: LinearLayout = findViewById<View>(R.id.linear_layout_noimage) as LinearLayout
+            val textView = TextView(this)
+            textView.text = "No Image Found"
```

Comment:
Use string localization for the text.

Reply:
how can I translate this string to all possible languages, translation is done by other capable contributors, creating other issues in the repo , and moreover not all the strings have been localized yet , its still in progress

Reply:
Please check how localization is done in other parts of the app. It is the standard Android way, so if you are not familiar with it I recommend doing a tutorial to get the idea. If you find strings in the app UI that are not localized, please create an issue to flag them, thanks a lot! :-)